runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -128,6 +136,14 @@
      */
     public volatile boolean shutdown;
 
+    /**
+     * Indicates if SSL authentication has failed or not, in order to apply a backoff mechanism for
+     * exponential time reconnection.
+     */
+    private AtomicBoolean sslAuthenticationFailure;
```

Comment:
isSslAuthenticationFailed is more expressive (and then optionally remove the variable comment)